src/modules/NtpClient.rb

```diff
@@ -33,7 +33,7 @@ class NtpClientClass < Module
     # @see #http://www.pool.ntp.org/
     RANDOM_POOL_NTP_SERVERS = ["0.pool.ntp.org", "1.pool.ntp.org", "2.pool.ntp.org"].freeze
 
-    NTP_FILE = "/etc/chrony.conf".freeze
+    NTP_FILE = "/etc/chrony.d/pool.conf".freeze
```

Comment:
So, the pool.conf will not contain only pool.conf but also servers and probably other entries there isn't it? Maybe we should have our own config file as we do with sysctl config.

Reply:
either we can split it into "server.conf" and "pool.conf", however it sounds a bit overkill to me ...  or have some neutral "sources.conf" with all together as it is now (but with better name). I'm not good in naming so I didn't spend much time in the area yet and I'm open to name suggestions ;-)

Reply:
one benefit if we write it to pool.conf is that we need to deal with one provided by default by chrony

Reply:
To summarize what I got from release manager(s)
- in general fine with the change
- wants to keep pool.conf in chrony-pool-* package because of image based installations
- fine with overwriting pool.conf from chrony-pool-* in case of yast driven installation
- having server directive in pool.conf was not problem

So, basically the patch should, i believe, fulfill the above.